src/hooks/admin/use-localization-toolkit/useLocalizationToolkit.ts

```diff
@@ -0,0 +1,58 @@
+import { useCallback, useEffect, useState } from 'react';
+import { LocalizationLanguage, TranslationStatusFilter } from '../../../types/common/language';
+import { localizationLanguagesDataFetch } from '../../../services/api/public/localization/languages/languages-api';
+import axios from 'axios';
+import { COMMON_TEXT_ADMIN } from '../../../const/admin/common';
+import { DEFAULT_LOCALE } from '../../../const/common/locales';
+
+export interface UseLocalizationToolkitProps {
+    setErrorState: (message: string, type: any) => void;
```

Comment:
avoid `any`.
probably like this? double check it
```suggestion
    setErrorState: (message: string, type: 'languages') => void;
```

Reply:
fix it